src/App.scss

```diff
@@ -164,28 +164,47 @@ body {
     grid-column-end: 3;
     grid-row-start: 2;
     grid-row-end: 4;
-    display: grid;
+    display: flex;
+    flex-direction: column;
     grid-template-rows: min-content min-content auto;
 }
 
 .App__command-palette-command-container {
     background-color: #30444E;
     border-radius: 3px;
-    width: 100%;
     height: 100%;
+    width: 100%;
     overflow-y: auto;
     grid-row-start: 3;
     grid-row-end: 4;
     padding-top: 1rem;
 }
 
-.App__command-palette-commands {
-    width: 6rem;
+.App__command-palette-section-heading {
+    border-bottom: 1px solid #818A98;
+    color: #F1AE5B;
+    font-size: .9rem;
+    text-align: left;
+    padding-bottom: .6rem;
```

Comment:
We can remove this padding-bottom and padding-left from the heading h3 after wrap them in a container.

Reply:
We also need to apply margin-top and margin-bottom 0 to the heading h3, because default h3 applies some margin-top and margin-bottom.